WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionSheetViewController.swift

```diff
@@ -259,7 +261,7 @@ class ReaderSortingOptionViewController: UIViewController {
 
 extension ReaderSortingOptionViewController: Accessible {
     func prepareForVoiceOver() {
-        gripButton.accessibilityLabel = NSLocalizedString("Dismiss", comment: "Accessibility label for the grip button which is used to dismiss the dialog.")
+        gripButton.isAccessibilityElement = false
```

Comment:
@Gio2018 I have set the sheet handle not to be accessibility element, so we do not have VoiceOver for it. Let me know if I should remove the logic for dissmissing the sheet if user taps it (as mentioned on slack)